ui/src/seminars/components/SearchPage.jsx

```diff
@@ -14,7 +15,7 @@ function SearchPage() {
     <>
       <DocumentHead title={TITLE} description={META_DESCRIPTION} />
       <Row>
-        <Col xs={24} lg={22} xl={20} xxl={18}>
+        <Col {...SEARCH_PAGE_COL_SIZE_WITH_FACETS}>
```

Comment:
Same here. Btw that can also be a container in case we want to utilize your funciton.

Reply:
replied above